### PR TITLE
VITS Emotional training support

### PR DESCRIPTION
**Pull request by Edresson**

### CODE REVIEW

File: TTS/tts/models/vits.py [hunk 1]
```diff
@@ -529,14 +641,73 @@ class VitsArgs(Coqpit):
...
+    use_external_emotions_embeddings: bool = False
+    emotions_ids_file: str = None
+    external_emotions_embs_file: str = None
+    emotion_embedding_dim: int = 0
+    num_emotions: int = 0
+    use_text_enc_spk_reversal_classifier: bool = False
+    use_text_enc_emo_classifier: bool = False
+
+    # emotion and speaker embedding squeezer
+    use_emotion_embedding_squeezer: bool = False
```
Comment (erogol) [inline TTS/tts/models/vits.py:656]: These are in general called ```projection``` not ```squeezer```. There are also squeeze layers but they are different.

**Comment by erogol** [comment]
Is there any diff between emotion and speaker managers? Maybe we should call them attribute_manager and use it for all.

**Comment by WeberJulian** [comment]
> Maybe we should call them attribute_manager and use it for all.

I agree with that, especially if we continue to add more, same with language btw.

**Comment by Edresson** [comment]
> Is there any diff between emotion and speaker managers? Maybe we should call them attribute_manager and use it for all.

Yeah agreed, as we discussed, let's keep it this way for now, but when we will add new managers we need to try to group then.

**Comment by scientificCommunity** [comment]
hi! would you tell me how to use this extension? Thank you very much!

**Pull request closed without merging**